bundles/org.eclipse.equinox.p2.repository.tools/build.properties

```diff
@@ -25,4 +25,5 @@ source.lib/repository-tools-ant.jar = src_ant/
 output.lib/repository-tools-ant.jar = bin_ant/
 jars.extra.classpath=platform:/plugin/org.apache.ant/lib/ant.jar,\
                      platform:/plugin/org.eclipse.equinox.p2.jarprocessor/lib/jarprocessor-ant.jar
+pom.model.property.javadoc.excludePackageNames = org.eclipse.equinox.p2.internal.repository.tools.tasks
```

Comment:
@laeubi  @HannesWell 

I tried to use this to eliminate the last Javadoc errors.

But still I get this:
```
16:32:38  [ERROR] MavenReportException: Error while generating Javadoc: 
16:32:38  Exit code: 1
16:32:38  /home/jenkins/agent/workspace/p2_PR-546/bundles/org.eclipse.equinox.p2.repository.tools/src_ant/org/eclipse/equinox/p2/internal/repository/tools/tasks/ProcessRepoTask.java:25: error: package org.eclipse.equinox.internal.p2.jarprocessor.ant does not exist
16:32:38  import org.eclipse.equinox.internal.p2.jarprocessor.ant.JarProcessorTask;
16:32:38                                                         ^
16:32:38  /home/jenkins/agent/workspace/p2_PR-546/bundles/org.eclipse.equinox.p2.repository.tools/src_ant/org/eclipse/equinox/p2/internal/repository/tools/tasks/ProcessRepoTask.java:64: error: cannot find symbol
16:32:38  	private JarProcessorTask jarProcessor = null;
16:32:38  	        ^
16:32:38    symbol:   class JarProcessorTask
16:32:38    location: class ProcessRepoTask
16:32:38  2 errors
16:32:38  Command line was: /opt/tools/java/temurin/jdk-17/jdk-17.0.10+7/bin/javadoc -J-Duser.language= -J-Duser.country= @options @packages
```
Any idea?

Reply:
You also have to prepare this in the javadoc plugin config:
https://github.com/eclipse-pde/eclipse.pde/blob/b00c27d7366cd34c764d3046f32ed808d301bbfd/pom.xml#L62-L68

Reply:
We should consider to add this to the eclipse-parent so that not each project that needs it has to duplicate it.